sharding-repl-cache/scripts/mongo-init.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+###
+# Инициализируем бд
+###
+
+docker compose exec -i configSrv mongosh --port 27016 <<EOF
```

Comment:
Вот где была ошибка, из-за этого не настраивался конфиг сервер и к нему роутер не мог подключиться

```suggestion
docker exec -i configSrv mongosh --port 27016 <<EOF
```